### ISSUE TITLE
[HL1S] [Steampipe] Map c1a2a Bug 66

### ISSUE THREAD

**Issue by Pinhead-alejandro** [Theuaredead/Half-Life-Source-Update#69]
Well when you start this map, and you are placed close to this window:

![2015-07-14_00042](https://cloud.githubusercontent.com/assets/13278870/8681228/9717820c-2a66-11e5-994e-b380975b524d.jpg)

This scientist should die:

![2015-07-14_00036](https://cloud.githubusercontent.com/assets/13278870/8681253/bb93baba-2a66-11e5-93fc-0831059eb329.jpg)

At less, he die in Half-Life. However, this could be a fixed bug, fixed to make possible to the player see the whole situation, and avoid the death of the second scientist that you can actually find inside the dark room showed in the first screenshot. In Half-Life this second scientist dies a lot of times, and it is sad because you can't hear his sentence if he dies. This is a difference between Half-Life and Half-Life: Source that maybe we shoudn't change... but maybe, if it is a bug we should. Make your choise.

**Comment by Theuaredead** [comment]
I've always been confused on why there were two scientists, in both games.


However I'm almost certain his faith is that he should die.

**Comment by Theuaredead** [comment]
though, I think the issue he doesn't die is because the relations to the NPCs don't start until the NPCs are aware the player is there.

Though, by luck, I managed to see him run into the barnacle (which isn't how he dies in Half-Life)

![2015-07-14_00001](https://cloud.githubusercontent.com/assets/5455744/8682762/8b337aec-2a23-11e5-9b21-c843611caad5.jpg)

**Comment by Pinhead-alejandro** [comment]
True, that is another differnce, but, I would like to note another aspect: In Half-Life, the second scientist usually dies just because he is seen by the Vortigaunt that kills the first scientist.
